frontend/src/routes/ClusterManagement/Clusters/CreateCluster/controlData/ControlDataAWS.js

```diff
@@ -616,14 +621,16 @@ const controlDataAWS = [
 
     ////////////////////////////////////////////////////////////////////////////////////
     ///////////////////////  node(machine) pools  /////////////////////////////////////
+    {
+        id: 'mpoolsStep',
+        type: 'step',
+        title: 'Master cluster pool',
```

Comment:
might get confusing with cluster pools?

Reply:
Can we resolve this today and get code merged?